src/main/kotlin/no/nav/amt/person/service/nav_bruker/NavBrukerService.kt

```diff
@@ -117,6 +121,27 @@ class NavBrukerService(
 		}
 	}
 
+	fun oppdaterOppfolgingsperiode(navBrukerId: UUID, oppfolgingsperiode: Oppfolgingsperiode) {
```

Comment:
Jeg tror denne kan gjøres enklere

```suggestion
	fun oppdaterOppfolgingsperiode(navBrukerId: UUID, oppfolgingsperiode: Oppfolgingsperiode) {
		val bruker = repository.get(navBrukerId).toModel()
		val oppfolgingsperioder = bruker.oppfolgingsperioder.filter { it.id != oppfolgingsperiode.id }.plus(oppfolgingsperiode)

		if (oppfolgingsperioder.toSet() != bruker.oppfolgingsperioder.toSet()) {
			upsert(bruker.copy(oppfolgingsperioder = oppfolgingsperioder))
		}
	}
```

Reply:
Det her var mye bedre!